packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2492,25 +2492,24 @@ export class LitNodeClientNodeJs extends LitCore {
 
     let siweMessage;
 
-    if (params?.capability && params?.litResource) {
+    if (params?.resourceAbilityRequests) {
+      const resources = params.resourceAbilityRequests.map((r) => r.resource);
```

Comment:
@joshLong145 this would allow users to only pass the `resourceAbilityRequests` once to the `getSessionSigs` function and also allowing multiple resources instead of a single resource previously

Reply:
Makes sense, do we need to order them such that the Capacity Increase request is always index `0`?